react/Header/Header.demo.js

```diff
@@ -80,6 +80,33 @@ export default {
         })
       }))
     },
+    {
+      label: 'New Tab',
+      type: 'radio',
+      states: [
+        undefined,
+        null,
+        'Job Search',
+        '$150k+ Jobs',
+        'Profile',
+        'Saved & Applied Jobs',
+        'Recommended Jobs',
+        'Company Reviews',
+        'Advice & Tips'
+      ].map(newBadgeTab => {
+        let label = newBadgeTab;
+        if (!newBadgeTab) {
+          label = newBadgeTab === undefined ? 'Default' : 'None';
```

Comment:
Sorry, one last nitpick—could this say `Default "new" badge` and `No "new" badge`? The dropdowns aren't labeled, so the default value needs to act as a descriptor too, otherwise it just says "Default" and users won't know what it's referring to.